source/parser/function.civet

```diff
@@ -1175,12 +1175,20 @@ function expressionizeIteration(exp: IterationExpression): void
     iterationDefaultBody statement
 
     assignResults block, (node) =>
-      type: "YieldExpression"
-      expression: node
-      children:
-        . type: "Yield"
-          token: "yield "
-        . node
+      let star: ASTNode
```

Comment:
Shouldn't this be `ASTNode | undefined`, since it's never set for non-spread expressions?